### PR TITLE
Jessica Yang add to team

### PR DESCRIPTION
**Pull request by jess94yng**

### CODE REVIEW

File: _data/team.yml [hunk 1]
```diff
@@ -116,6 +116,13 @@ people:
   position: Bioinformatics Specialist, UHN
   team: implementation  
   
+  - name: Jessica Yang
+  lastname: Yang
```
Comment (ljdursi) [inline _data/team.yml]: For YAML, indentation matters - please add spaces for "lastname", "firstname", etc to line up with "name".  Once that's done it should work!

**Review by ljdursi — CHANGES_REQUESTED** [review]
Let's see how the build looks after the indentation fix

File: _data/team.yml (hunk 1 above)
Comment (jess94yng) [reply _data/team.yml]: ok i've just fixed the indentation, please let me know if it still doesnt work!

**Comment by ljdursi** [comment]
Sorry - could you "out-dent" the `-name: Jessica Yang` line so it matches the zero-indent of Daisie's line above?  From

```
- name: Daisie Huang
  lastname: Huang
[...]

  - name: Jessica Yang
  lastname: Yang        
[...]
  
 - name: Pierre-Étienne Jacques
   lastname: Jacques
 ``` 
 
 to 
 ```
- name: Daisie Huang
  lastname: Huang
[...]

- name: Jessica Yang
  lastname: Yang        
[...]
  
 - name: Pierre-Étienne Jacques
   lastname: Jacques
 ```

**Review by ljdursi — APPROVED** [review]

**Comment by ljdursi** [comment]
Terrific - looks good, thanks!

**Pull request merged**